### PR TITLE
Re-test Shibuya and Rocstar with Astar v5.0.1 and OAK v1.8.1

### PR DESCRIPTION
**Pull request by chrisli30**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,9 +165,9 @@ Task: {
 ### Pre-requisites
 | Chain      | Version | Commit hash |
 | :---        |    :----:   |          ---: |
-| Polkadot      | [0.9.29](https://github.com/paritytech/polkadot/releases/tag/v0.9.29)       |		[94078b4](https://github.com/paritytech/polkadot/commit/94078b44fb6c9767bf60ffcaaa3be40681be5a76)  |
-| OAK-blockchain   | [1.8.0](https://github.com/OAK-Foundation/OAK-blockchain/releases/tag/untagged-2aecbd94ab4bcde05657)     |	[d04462](https://github.com/OAK-Foundation/OAK-blockchain/pull/328/commits/d044a62825746e6dd8b7593a6c7dfb9eefcac308)  |
-| Astar | x   | [523c067](https://github.com/AstarNetwork/Astar/commit/523c06798a08189a3ea20f790b83cd4ae602c579)        |
+| Polkadot      | [v0.9.39](https://github.com/paritytech/polkadot/releases/tag/v0.9.39)      |
```
Comment (chrisli30) [inline README.md:168]: v0.9.39 is Astar’s v5.0.1 dependency. Our dependency v0.9.29 might or might not work with XCM v3. We should try to upgrade polkadot.

File: src/astar/rocstar.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ const scheduleTask = async ({
         { Recurring: { frequency: TASK_FREQUENCY, nextExecutionTime } },
         // { Fixed: { executionTimes: [0] } },
         shibuyaHelper.config.paraId,
-        4,
```
Comment (chrisli30) [inline src/astar/rocstar.js]: @imstar15  DO NOT hard-code value in code!

File: src/astar/rocstar.js [hunk 3]
```diff
@@ -49,7 +49,8 @@ const scheduleTask = async ({
         { Recurring: { frequency: TASK_FREQUENCY, nextExecutionTime } },
         // { Fixed: { executionTimes: [0] } },
         shibuyaHelper.config.paraId,
-        0,
+        paraTokenIdOnTuring,
+        { V1: { parents: 1, interior: { X1: { Parachain: shibuyaHelper.config.paraId } } } },
         encodedCallData,
         encodedCallWeight,
     );
```
Comment (chrisli30) [inline src/astar/rocstar.js:56]: Don’t you need to apply the same change to remove the below line? @imstar15 
```
    const taskViaProxy = turingHelper.api.tx.proxy.proxy(turingAddress, 'Any', taskExtrinsic);
```

File: src/astar/shiden.js [hunk 4]
```diff
@@ -50,7 +50,8 @@ const scheduleTask = async ({
         { Recurring: { frequency: TASK_FREQUENCY, nextExecutionTime } },
         // { Fixed: { executionTimes: [0] } },
         shibuyaHelper.config.paraId,
-        0,
+        paraTokenIdOnTuring,
+        { V1: { parents: 1, interior: { X1: { Parachain: shibuyaHelper.config.paraId } } } },
         encodedCallData,
         encodedCallWeight,
     );
```
Comment (chrisli30) [inline src/astar/shiden.js:57]: Don’t you need to apply the same change to remove the below line? @imstar15 
```
    const taskViaProxy = turingHelper.api.tx.proxy.proxy(turingAddress, 'Any', taskExtrinsic);
```

File: src/astar/rocstar.js (hunk 3 above)
Comment (imstar15) [reply src/astar/rocstar.js:56]: Okay. Repalced scheduleXcmpTask with scheduleXcmpTaskThroughTask in rocstar.

File: src/astar/shiden.js (hunk 4 above)
Comment (imstar15) [reply src/astar/shiden.js:57]: Okay. Repalced scheduleXcmpTask with scheduleXcmpTaskThroughTask in shiden.

**Comment by chrisli30** [comment]
The comments are addressed and the PR looks good. Good to merge.

**Pull request merged**